protos/escrow/escrow.proto

```diff
@@ -72,6 +73,7 @@ message SignedEscrowContract {
 
 message EscrowContractRequest {
   repeated SignedEscrowContract contract = 1;
+  ledger.SignedChannelCommit buyer_channel = 4; // ledger channel commit from buyer to escrow
```

Comment:
Not fully sure whether it is need, seems escrow can construct the buyer_channel_state without buyer prepare this.

Reply:
This is for buyer to sign the channel commit then escrow can create ledger channel, because ledger don't have buyer's private key.
SignedChannelCommit {
  ChannelCommit channel = 1;
  bytes signature = 2;
}